contrib/opentimelineio_contrib/adapters/extern_rv.py

```diff
@@ -288,20 +301,25 @@ def _write_item(it, to_session, track_kind=None):
             )
         )
 
-    # because OTIO has no global concept of FPS, the rate of the duration is
-    # used as the rate for the range of the source.
-    # RationalTime.value_rescaled_to returns the time value of the object in
-    # time rate of the argument.
-    src.setCutIn(
-        range_to_read.start_time.value_rescaled_to(
+    _in = _out = None
```

Comment:
nitpick: these could be called `in_frame` and `out_frame`, the `_` prefix feels a little funny to me.

Reply:
agree, on it now